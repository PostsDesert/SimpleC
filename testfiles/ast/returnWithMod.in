int gcd2(int a, int b) {
return gcd2(b, a % b);
}