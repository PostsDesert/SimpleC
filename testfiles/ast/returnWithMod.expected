Program
  List
    FuncDesc
      IdDecl ID="gcd2"
      List
        IdDecl ID="a"
        IdDecl ID="b"
      Block
        List
          ReturnStmt
            FuncCall
              IdUse ID="gcd2"
              List
                IdUse ID="b"
                Mod
                  IdUse ID="a"
                  IdUse ID="b"
